clang/test/Sema/builtin-cpu-supports.c

```diff
@@ -27,7 +27,7 @@ int main(void) {
   (void)__builtin_cpu_supports("x86-64-v4");
   (void)__builtin_cpu_supports("x86-64-v5"); // expected-warning {{invalid cpu feature string for builtin}}
 #else
-  if (__builtin_cpu_supports("neon")) // expected-warning {{invalid cpu feature string for builtin}}
```

Comment:
If we want the target attribute to accept it then we should accept it here too, otherwise be consistent and reject in both cases.